starknet_py/tests/e2e/client/full_node_test.py

```diff
@@ -5,58 +5,6 @@
 from starkware.starknet.public.abi import get_storage_var_address
 
 from starknet_py.net.client_errors import ClientError
-from starknet_py.net.client_models import (
-    DeployTransaction,
-    DeployedContract,
-)
-
-
-@pytest.mark.run_on_devnet
-@pytest.mark.asyncio
-async def test_node_get_transaction_by_block_id_and_index(
-    block_with_deploy_number,
-    deploy_transaction_hash,
-    contract_address,
-    full_node_client,
-    class_hash,
-):
-    tx = await full_node_client.get_transaction_by_block_id(
-        block_number=block_with_deploy_number, index=0
-    )
-
-    assert tx == DeployTransaction(
-        hash=deploy_transaction_hash,
-        contract_address=contract_address,
-        constructor_calldata=[],
-        max_fee=0,
-        signature=[],
-        class_hash=class_hash,
-        version=0,
-    )
```

Comment:
I think we should have a test like this, just not with DeployTransaction.

Reply:
We already have one, a few lines below :)